verde/utils.py

```diff
@@ -254,6 +255,72 @@ def grid_to_table(grid):
     return data
 
 
+def load_surfer(fname, dtype='float64'):
+    """
+    Read data from a Surfer ASCII grid file.
+
+    Surfer is a contouring, griding and surface mapping software
+    from GoldenSoftware. The names and logos for Surfer and Golden
+    Software are registered trademarks of Golden Software, Inc.
+
+    http://www.goldensoftware.com/products/surfer
+
+    This feature opens a Surfer grid file, masks any NaNs in the data,
+    and outputs a `xarray.DataArray` that contains easting and northing
+    coordinates, data values, and associated file metadata in the
+    DataArray attributes.
+
+    Parameters
+    ----------
+    fname : str
+        Name or path of the Surfer grid file
+    dtype : str or numpy dtype
+        The type of variable used for the data. Default is ``float64``. Use
+        ``float32`` if the data are large and precision is not an issue.
+    Returns
+    ----------
+    data : :class:`xarray.DataArray`
+        A 2D grid with the data variables.
+    """
+    # Surfer ASCII grid structure
+    # DSAA            Surfer ASCII GRD ID
+    # nCols nRows     number of columns and rows
+    # xMin xMax       X min max
+    # yMin yMax       Y min max
+    # zMin zMax       Z min max
+    # z11 z21 z31 ... List of Z values
+    with open(fname) as input_file:
+        # DSAA is a Surfer ASCII GRD ID (discard it for now)
+        grd_id = input_file.readline()
```

Comment:
```suggestion
        grd_id = input_file.readline().strip()
```

To remove any extra spaces or newlines.

Reply:
that's the call I was looking for but couldn't remember...